README.md

```diff
@@ -1,21 +1,36 @@
 <img src="https://github.com/chanzuckerberg/starfish/raw/master/design/logo.png" width="250">
 
-The goal of *starfish* is to **prototype** a reference pipeline for the analysis of image-based transcriptomics data that works across technologies. This is a **work in progress** and will be developed in the open. 
+The goal of *starfish* is to **prototype** a reference pipeline for the analysis of image-based transcriptomics data that works across technologies. This is a **work in progress** and will be developed in the open.
 
 ## concept
 See this [document](https://docs.google.com/document/d/1IHIngoMKr-Tnft2xOI3Q-5rL3GSX2E3PnJrpsOX5ZWs/edit?usp=sharing) for details. The diagram below describes the core pipeline components and associated file manifests that this package plans to standardize and implement.
 
 ![alt text](https://github.com/chanzuckerberg/starfish/raw/master/design/pipeline-diagram.png "candidate pipeline")
 
-## usage
-Install the starfish package:
+## installation
+Starfish supports python 3.6 and above. To Install the starfish package, first verify that your python version is compatible. You can check this with pip, which may be called `pip` or `pip3` depending on how you installed python.
+
+The output should look similar to this:
+```
+% pip3 --version
+pip 10.0.1 from /usr/local/lib/python3.6/site-packages/pip (python 3.6)
+```
+
+While not required, you may wish to set up a [virtualenv](https://virtualenv.pypa.io/en/stable/). To do this, execute:
 ```
-% pip install starfish
+% python -m venv .venv
 ```
 
+Install the starfish module in edit-mode and all the dependencies for starfish:
+```
+% pip install -e .
+% pip install -r REQUIREMENTS.txt
```

Comment:
Is this second one required?  My observations seem to suggest that `pip install -e .` is enough.